.github/workflows/test-linux-gpu.yml

```diff
@@ -0,0 +1,72 @@
+name: Unit-tests on Linux GPU
+
+on:
+  pull_request:
+  push:
+    branches:
+      - nightly
+      - main
+      - release/*
+  workflow_dispatch:
+
+env:
+  CHANNEL: "nightly"
```

Comment:
Is there a advantage of defining the env var here?

Reply:
Just following the pattern from other workflows. If you suggest fixing / cleanup can we do so to all of them?